.bumpversion.cfg

```diff
@@ -0,0 +1,18 @@
+[bumpversion]
+current_version = 0.12.0dev
+commit = True
+
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?P<release>.*)?
+serialize = 
+	{major}.{minor}.{patch}{release}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
+optional_value = placeholder
```

Comment:
What does the optional_value do?

Reply:
See this: https://github.com/peritus/bumpversion/issues/128. Basically a hack to allow a boolean option (dev or not).

Reply:
Honestly though I don't fully grock this...I messed around with a few options before I stumbled on this as a solution.